clientd3d/d3drender.c

```diff
@@ -5275,6 +5266,14 @@ void D3DRenderFontInit(font_3d *pFont, HFONT hFont)
    DeleteDC(hDC);
 }
 
+// Adjust background overlay UV coordinates to mitigate texture bleeding
+void SetOverlayTextureCoordinates(float* u, float* v) {

```

Comment:
Is this being done every frame?  Might it make sense to do it only once, when the overlay is created?

Also, this random epsilon is not very satisfying.  Is there perhaps a texture clamping mode that could be used instead?